src/routes/safe/components/Balances/SendModal/screens/ReviewSendFundsTx/index.tsx

```diff
@@ -100,6 +100,7 @@ const ReviewSendFundsTx = ({ onClose, onPrev, tx }: ReviewTxProps): React.ReactE
   const data = useTxData(isSendingNativeToken, tx.amount, tx.recipientAddress, txToken)
   const [manualSafeTxGas, setManualSafeTxGas] = useState(0)
   const [manualGasPrice, setManualGasPrice] = useState<string | undefined>()
+  const [manualGasLimit, setManualGasLimit] = useState<string | undefined>()
```

Comment:
Not to do it now.
But we might refactor `useEstimateTransactionGas` and for `safeTxGas`. `gasPrice` and `gasLimit` instead of saving the raw value, we could save an object like: `{isManual: boolean, value: string}`. If we do so, I think we can remove the manual values everywhere that is using the hook.

Reply:
For sure we should think about an improvement for this. In general how we handle manual or automatic values is one of the biggest weaknesses we have here, as it requires to be changed in many places in order to be updated